src/__tests__/account.test.ts

```diff
@@ -0,0 +1,80 @@
+import { utils, providers, Wallet } from 'ethers';
+import { signMessage, getWallet, signTransaction, signHash } from '../account';
+import { tokenFaucet } from '../contract';
+import { MumbaiNetworkConfig } from '../network_config/network_config';
+
+// mock native code, just testing signing function
+
+jest.mock('react-native', () => {
+  const mnemonic =
+    'bronze adjust crane guard crater merry village wealth smoke exit woman cabbage';
+  const pk =
+    '0x8666ebf0f4d397955c55932642fb9dd97fb60a2df121a9d2a39f7f1930958ca3';
+
+  return {
+    NativeModules: {
+      RlyNetworkMobileSdk: {
+        getMnemonic: () => Promise.resolve(mnemonic),
+        generateMnemonic: () => Promise.resolve(mnemonic),
+        saveMnemonic: () => Promise.resolve(),
+        deleteMnemonic: () => Promise.resolve(),
+        getPrivateKeyFromMnemonic: () => Promise.resolve(pk),
+      },
+    },
+    Platform: {
+      select: jest.fn(),
+    },
+  };
+});
+
+test('sign message', async () => {
+  const message = 'test message';
+  const wallet = await getWallet();
+  const signature = await signMessage(message);
+  const sender = utils.verifyMessage(message, signature);
+  expect(wallet?.address).toEqual(sender);
+});
+
+test('sign transaction', async () => {
+  const wallet = await getWallet();
+
+  const provider = new providers.JsonRpcProvider(
+    MumbaiNetworkConfig.gsn.rpcUrl
```

Comment:
Would be nice to use a dummy / mock network here to avoid the need to hit external network to run these tests. Alternatively might be nice to at least flag them as integration tests, so we can prioritize running them in a different setup from more isolated unit tests.

Reply:
yeah the network here really isn't required, the method we're calling on the contract `populateTransaction` should work independent of network, think the best solution is to just hardcode the transaction object in the test.